build.ps1

```diff
@@ -1,13 +1,63 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+<#
```

Comment:
Separate change please, since it has nothing to do with the NuGet package.

Reply:
Sure thing

---
In reply to: [333685945](https://github.com/microsoft/CLRInstrumentationEngine/pull/150#discussion_r333685945) [](ancestors = 333685945)